src/repository.cpp

```diff
@@ -897,7 +924,8 @@ void FastImportRepository::Transaction::commit()
     Q_ASSERT(mark < repository->next_file_mark - 1);
 
     // create the commit message
-    QByteArray message = log;
+    // Optionally filter the log message through external program
+    QByteArray message = repository->msgFilter(log);
```

Comment:
Good suggestion.

I moved the filter process to after the addition of the meta data so that could also be filtered / modified.